### PR TITLE
Issue50

### PR DESCRIPTION
**Pull request by vicgabal**
I have finished the gulpfile. It has tasks to compile and watrch html, jsx, scss files in FRONTEND/SOURCE and put them in FRONTEND/BUILD

I have deciced not to use for now the gulp.babel plugin

I want to test that it works. Does it look ok to you? 

The latest commits changing this file are all in my feature branch "issue50"

I am going to create now my index.html and app.jsx and work on the React components of the landing page

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+var browserify = require('browserify');
+var vinylSourceStream = require('vinyl-source-stream');
+var vinylBuffer = require('vinyl-buffer');
+var reactify = require('reactify');
+var htmlmin = require('gulp-htmlmin');
+var sass = require('gulp-sass');
+
+gulp.task('build-js', function() {
+  return browserify('./source/js/app.jsx')
+        .transform(reactify)
```
Comment (fedosejev) [inline gulpfile.js]: Update to `babelify`.

File: gulpfile.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+});
+
+gulp.task('compile-sass', function () {
+  return gulp.src('./sass/**/*.scss')
+    	.pipe(sass().on('error', sass.logError))
+    	.pipe(gulp.dest('.build/css'));
+});
+
+gulp.task('watchers', function() {
+	gulp.watch('./frontend/**/*.jsx', ['build-js']);
```
Comment (fedosejev) [inline gulpfile.js]: You can use `*.{jsx, js}` to watch JSX _and_ JS.

**Comment by fedosejev** [comment]
Create `default` task as well, so that you could simply run `gulp`.

**Comment by vicgabal** [comment]
+ build paths corrected with ./
+ glob search for jsx and js grouped as .{jsx,js}
+ added a default task with an array of tasks that will run when comand gulp is executed in Terminal
+ since plugin reactify is deprecated, I have changed it for babelify: 
 + https://www.npmjs.com/package/babelify
 + npm install --save-dev babelify
 + npm install --save-dev babel-preset-es2015 babel-preset-react

**Comment by vicgabal** [comment]
There is an app.jsx file in place but I get the error: 

Cannot find module '/Users/VictorG/gits/tiy-final-project/source/js/app.jsx' from '/Users/VictorG/gits/tiy-final-project'

**Comment by vicgabal** [comment]
Error fixed. Missed to indicate that the parent folder is FRONTEND 
Also added React and React.DOM plugins, that were missed earlier

**Pull request merged**